tests/WebAppCallsWebApiCallsGraph/TodoListService/Controllers/LongRunningProcessContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.IdentityModel.Tokens.Jwt;
+using Microsoft.AspNetCore.Http;
+
+namespace TodoListService
+{
+    public class LongRunningProcessContext : IDisposable
+    {
+        JwtSecurityToken _savedToken;
+        HttpContext _httpContext;
+
+        internal LongRunningProcessContext(HttpContext httpContext, JwtSecurityToken tokenToUse)
+        {
+            _httpContext = httpContext;
+            _savedToken = httpContext.Items["JwtSecurityTokenUsedToCallWebAPI"] as JwtSecurityToken;
```

Comment:
Too bad we cannot use (as they are internal).
https://github.com/AzureAD/microsoft-identity-web/blob/800b833ddf788032ae885807dbbceb7581db8852/src/Microsoft.Identity.Web/HttpContextExtensions.cs#L17

and

https://github.com/AzureAD/microsoft-identity-web/blob/800b833ddf788032ae885807dbbceb7581db8852/src/Microsoft.Identity.Web/HttpContextExtensions.cs#L31

We also have an issue where the customers would like us to Store the incoming token somewhere.

Reply:
agree...should we open an issue?